js/src/WebsocketConnection/index.d.ts

```diff
@@ -17,6 +17,12 @@ export interface WebsocketSession {
   // Cancels the subscription.
   unsubscribe(info: SubscriberInfo): Promise<void>;
   close(): Promise<void>;
+
+  /**
+   * @param payload The binary data to send
+   * @returns The id assigned to the binary attachment
+   */
+  addAttachment(payload: Blob): number
```

Comment:
I'm not familiar with the JS side, but are you sure it is a number rather than a string that `id` such attachment?

Reply:
You're right; one loc = one error, sorry. :-)